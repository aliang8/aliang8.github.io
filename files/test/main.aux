\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\zref@newlabel[2]{}
\bbl@cs{beforestart}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{none/global//global/global}
\providecommand \oddpage@label [2]{}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\babel@aux{english}{}
\pgfsyspdfmark {pgfid1}{4661699}{25290308}
\@writefile{toc}{\contentsline {chapter}{\numberline {1}Reinforcement Learning for Sparse-Reward Object-Interaction Tasks in First-person Simulated 3D Environments}{1}{chapter.1}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{loa}{\addvspace {10\p@ }}
\@writefile{toc}{\contentsline {chapter}{\numberline {2}Fellowship Project Proposal}{2}{chapter.2}\protected@file@percent }
\@writefile{lof}{\addvspace {10\p@ }}
\@writefile{lot}{\addvspace {10\p@ }}
\@writefile{loa}{\addvspace {10\p@ }}
