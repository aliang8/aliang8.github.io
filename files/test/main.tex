\documentclass[a4paper, 12pt]{report}

%%%%%%%%%%%%
% Packages %
%%%%%%%%%%%%

\usepackage[english]{babel}
\usepackage[noheader]{packages/sleek}
\usepackage{packages/sleek-title}
\usepackage{packages/sleek-theorems}
\usepackage{packages/sleek-listings}

%%%%%%%%%%%%%%
% Title-page %
%%%%%%%%%%%%%%

% \logo{./resources/pdf/logo.pdf}
% \institute{Random University}
% \faculty{Faculty of Whatever Sciences}
%\department{Department of Anything but Psychology}
\title{Writing Samples}
\subtitle{for application to Stanford Computer Science Ph.D program}
\author{Anthony \textsc{Liang}}
%\supervisor{Linus \textsc{Torvalds}}
%\context{Well, I was bored...}
\date{\today}

%%%%%%%%%%%%%%%%
% Bibliography %
%%%%%%%%%%%%%%%%

\addbibresource{./resources/bib/references.bib}

%%%%%%%%%%
% Others %
%%%%%%%%%%

\lstdefinestyle{latex}{
    language=TeX,
    style=default,
    %%%%%
    commentstyle=\ForestGreen,
    keywordstyle=\TrueBlue,
    stringstyle=\VeronicaPurple,
    emphstyle=\TrueBlue,
    %%%%%
    emph={LaTeX, usepackage, textit, textbf, textsc}
}

\FrameTBStyle{latex}
\usepackage[ruled,vlined]{algorithm2e}
\def\tbs{\textbackslash}

%%%%%%%%%%%%
% Document %
%%%%%%%%%%%%

\begin{document}
    \maketitle
    \romantableofcontents

    \chapter{Reinforcement Learning for Sparse-Reward Object-Interaction Tasks in First-person Simulated 3D Environments}
    \chapter{Fellowship Project Proposal}

\end{document}
