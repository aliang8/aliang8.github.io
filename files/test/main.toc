\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax 
\babel@toc {english}{}
\contentsline {chapter}{\numberline {1}Reinforcement Learning for Sparse-Reward Object-Interaction Tasks in First-person Simulated 3D Environments}{1}{chapter.1}%
\contentsline {chapter}{\numberline {2}Fellowship Project Proposal}{2}{chapter.2}%
