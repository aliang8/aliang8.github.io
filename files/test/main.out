\BOOKMARK [0][-]{chapter.1}{Reinforcement Learning for Sparse-Reward Object-Interaction Tasks in First-person Simulated 3D Environments}{}% 1
\BOOKMARK [0][-]{chapter.2}{Fellowship Project Proposal}{}% 2
